packages/protocol/migrationsConfig.js

```diff
@@ -202,6 +203,65 @@ const DefaultConfig = {
   },
 }
 
+const NetworkConfigs = {
+  testing: {
+    downtimeSlasher: {
+      slashableDowntime: 6,
+    },
+    election: {
+      minElectableValidators: '1',
+    },
+    epochRewards: {
+      frozen: false,
+    },
+    exchange: {
+      frozen: false,
+    },
+    goldToken: {
+      frozen: false,
+    },
+    stableToken: {
+      frozen: false,
+    },
+    reserve: {
+      initialBalance: 100000000,
+    },
+  },
+  baklava: {
+    blockchainParameters: {
+      minimumClientVersion: {
+        major: 0,
```

Comment:
Have we updated the version number in celo-blockchain?

Reply:
Yes with the block header deletion it is now `0.10.0`